components/engine_rocks/src/cf_options.rs

```diff
@@ -40,6 +40,23 @@ impl RocksCfOptions {
     pub fn into_raw(self) -> RawCfOptions {
         self.0
     }
+
+    pub fn set_flush_size(&mut self, f: usize) -> Result<()> {
```

Comment:
```suggestion
    pub fn set_flush_size(&mut self, flush_size: usize) -> Result<()> {
```

Reply:
`f` is used for consistency as other `set_flush_size` use `f`.